app/contracts/members/delete_contract.rb

```diff
@@ -27,15 +27,7 @@
 #++
 
 module Members
-  class DeleteContract < ::DeleteContract
+  class DeleteContract < DeleteBaseContract
```

Comment:
❓ what is the reason for not having this names something project specific. That way it would be more easily discernible from the `GlobalDeleteContract`

Reply:
I figured it would be easier to merge bugfixes back into `dev` later if I don't refactor class names that are already in use throughout the application.

However, I have now pushed a commit where the class names are normalized and I think it's much more readable, just like you suggested.